sretoolbox/container/image.py

```diff
@@ -113,18 +127,34 @@ def __init__(self, url, tag_override=None, username=None, password=None,
 
         self._cache_tags = None
         self._cache_manifest = None
+        self._cache_content_type = None
+
+    @property
+    def content_type(self):
+        """
+        Property to return the Content-Type header from the manifest retrieval.
+        It caches the result.
+        """
+        if self._cache_content_type is None:
+            _ = self.manifest
+
+        if self._cache_content_type is NO_SUCH_HEADER:
```

Comment:
no need for the `NO_SUCH_HEADER` really.. if you called `_ = self.manifest` and it is still `None` after that, it's time to raise the `HTTPError`.

Reply:
good call 😂

Reply:
fixed in b58b09b